schema_utils_test.py

```diff
@@ -253,8 +269,13 @@ def _validate_dict_keys(dict_to_check, required_keys, optional_keys):
     """Checks that all of the required keys, and possibly some of the optional
     keys, are in the given dict.
 
+    Args:
+        dict_to_check: dict. Dictionary for which keys are checked.
+        required_keys: list. Keys which are required to be in the dictionary.
+        optional_keys: list. Keys which are optionals in the dictionary.
+
     Raises:
-        AssertionError. The validation fails.
+        AssertionError. The validation fails if there are extra keys.
```

Comment:
AssertionError. The dict is missing required keys.
AssertionError. The dict contains extra keys.

Reply:
done